socorro/unittest/external/postgresql/test_graphics_devices.py

```diff
@@ -12,7 +12,7 @@
 from .unittestbase import PostgreSQLTestCase
 
 
-#==============================================================================
+# =============================================================================
```

Comment:
This is a larsism that is here to stay. Sure, we don't allow it in the webapp-django directory but for now I suggest you just allow it be by allowing flake8 to check and it should respect our [setup.cfg](https://github.com/mozilla/socorro/blob/master/setup.cfg).